### PR TITLE
Add information on testing changes locally via docker

### PR DESCRIPTION
**Pull request by rogeriopradoj**

### CODE REVIEW

**Comment by ramsey** [comment]
It looks like the latest version of etelej/jekyll was created over a year ago, so it doesn't include the latest versions of the gems in our `docs/Gemfile.lock` file.

**Comment by ramsey** [comment]
This might be an alternative image to use. It looks popular and appears to be kept up-to-date. Our `Gemfile.lock` specifies github-pages version 141, and their latest image is from 6 days ago, using github-pages version 137.

https://hub.docker.com/r/starefossen/github-pages/

**Comment by ramsey** [comment]
Unfortunately, I keep getting this error when trying to run the starefossen/github-pages image:

```
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/spec_set.rb:87:in `block in materialize': Could not find minitest-5.10.2 in any of the sources (Bundler::GemNotFound)
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/spec_set.rb:81:in `map!'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/spec_set.rb:81:in `materialize'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/definition.rb:159:in `specs'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/definition.rb:218:in `specs_for'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/definition.rb:207:in `requested_specs'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:109:in `block in definition_method'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:21:in `setup'
from /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler.rb:101:in `setup'
from /usr/local/bundle/gems/jekyll-3.4.3/lib/jekyll/plugin_manager.rb:36:in `require_from_bundler'
from /usr/local/bundle/gems/jekyll-3.4.3/exe/jekyll:9:in `<top (required)>'
from /usr/local/bundle/bin/jekyll:23:in `load'
from /usr/local/bundle/bin/jekyll:23:in `<main>'
```

**Comment by ramsey** [comment]
@rogeriopradoj After being unable to get any of the GitHub Pages Docker images I found to work, I created our own solution. :smile:

We now have a Docker image for previewing GitHub Pages sites here: https://hub.docker.com/r/phpcommunity/github-pages/

This works well. I've already tested it:

```
cd docs/
docker run -v "$PWD":/usr/src/app -p "4000:4000" phpcommunity/github-pages
```

Please update your directions here. Thanks!

File: README.md [hunk 1]
```diff
@@ -95,10 +95,38 @@ needed:
 
 ### Testing Changes Locally
 
-#### Configure Your Environment
+There are two ways to test changes to the `docs/` directory locally: via Docker
+or getting all the dependencies installed on your system.
 
-To test changes to the `docs/` directory locally, follow these steps to make
-sure you have the GitHub Pages dependencies installed:
+#### Via docker
```
Comment (ramsey) [inline README.md]: Add "(preferred)" next to this.

File: README.md [hunk 2]
```diff
@@ -95,10 +95,38 @@ needed:
...
+cd phptestfest.org/docs/
+```
+
+##### Previewing Changes
+
+To preview your changes, type the following into a command prompt from the
+`docs/` directory:
+
+``` bash
+docker run --rm -it -p 4000:4000 -v "$PWD":/app -w /app etelej/jekyll
```
Comment (ramsey) [inline README.md]: Change to `docker run -v "$PWD":/usr/src/app -p "4000:4000" phpcommunity/github-pages`

**Review by ramsey — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ramsey) [reply README.md]: You can also leave the `--rm -it` options in there, if you like.

File: README.md (hunk 1 above)
Comment (rogeriopradoj) [reply README.md]: Nice. I was willing to do this from beginning ;-).

File: README.md (hunk 2 above)
Comment (rogeriopradoj) [reply README.md]: > You can also leave the --rm -it options in there, if you like.

I prefer to remove the container after execution, so I will let --rm 😉 , ok?

**Comment by rogeriopradoj** [comment]
@ramsey please confirm it's fine now. Let me know if another adjust is needed.

Thanks!

**Review by ramsey — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**